pkg/proxy/util/utils.go

```diff
@@ -587,3 +587,16 @@ func RevertPorts(replacementPortsMap, originalPortsMap map[netutils.LocalPort]ne
 		}
 	}
 }
+
+// NodeContainsPodCIDR returns true if the node contains the passed cidr
```

Comment:
minor: Containes => Has (contains with CIDR indicates containment, e.g. 10.0.0.0/24 is contained inside 10.0.0.0/16)

Reply:
yep